mockdata/api/accounts_mgmt/v1/subscriptions.json

```diff
@@ -1038,6 +1038,162 @@
       "status": "Stale",
       "last_reconcile_date": "2019-06-24T18:30:15.127924Z"
     },
+    {
```

Comment:
please re-run the auto-generate script, to align this file with the contents of the subscription file, once updated.